rts/lra/test/pom.xml

```diff
@@ -27,7 +27,6 @@
 
   <properties>
 
-    <lra.coordinator.debug.params></lra.coordinator.debug.params>
```

Comment:
@mmusgrov do you know what happens when this is read in at: https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L359 - similar for the other removed ones. I presume it would be OK but I don't know. Also, I think it should go in a separate commit?

Reply:
It won't do anything because `<argument>${lra.coordinator.debug.params}</argument>` would then correspond to no argument.

I will push another commit that does not do any clean up (and only does what's necessary to fix the bug).

Reply:
My 2c:
- Initially, this and the other properties were defined in the [properties](https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L28-L55) section of the `lra-test` pom. Then, for some reason, they were moved in the profiles where they were actually utilised
- Removing these properties from the [properties](https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L28-L55) of the `lra-test` pom does not have any effect on the `properties` sections in profiles, i.e. things still work
- About `lra.coordinator.debug.params` in particular, I think it was a mistake: `lra.coordinator.debug.params` should have been defined in the `properties` section of the [this](https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L330) profile. For example, the `debug.lra.coordinator` [profile](https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L304) defines `debug.lra.coordinator` in its `properties` section.

Reply:
As a consequence, I think that we can go for either removing [lra.coordinator.debug.params](https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L359) from the `start.lra.coodinator.before.it.tests` profile or defining that property in the profile's `properties` section. I would +1 the first option

Reply:
> @mmusgrov do you know what happens when this is read in at: https://github.com/jbosstm/narayana/blob/master/rts/lra/test/pom.xml#L359 - similar for the other removed ones. I presume it would be OK but I don't know. Also, I think it should go in a separate commit?

@tomjenkinson I have reworked the PR to only do what is necessary to achieve the separation of the store between the two instances.

Reply:
OK, I see why all those properties were left blank now (and then activated through profiles)...but we should write a README to use this pom :rofl: